src/components/measureGroups/MeasureGroups.test.tsx

```diff
@@ -312,7 +312,7 @@ describe("Measure Groups Page", () => {
 
     // Update the definition
     const initialPopulationDescription = screen.getByTestId(
-      "initialPopulation-description"
+      "populations[0].description-description"
```

Comment:
If the testIds are being modified, please notify QA so they can update their automated tests.

Reply:
It's only modified against my previous test. Hasn't gone through QA for this yet